packages/angular/projects/clarity-adoption/src/app/migrations/4.0.0/icons/badge copy/icons.ng.html

```diff
@@ -0,0 +1,5 @@
+<clr-icon shape="user" class="has-badge has-badge--info"></clr-icon>
```

Comment:
`badge copy` ? can we don't have spaces in the file name/folders this could be in rare cases issue on windows and other tools if we don't build the path? correctly

Reply:
Seems like I made an accident copy. Fixed.